### DISCUSSION TITLE
2024 · 应届牛马到远程自由 | 愧怍

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#328] [General]
# 2024 · 应届牛马到远程自由 | 愧怍

面临着最后半年的大学生涯，与大多数大学生不同，我无时不在期盼着毕业这天的到来。步入职场进入了一家 AI 初创公司开始了"牛马"生活。最终我选择在国庆前裸辞，回到老家开始黑猴 play，却未曾料到，一份远程工作机会和一位新伙伴🐱悄然而至…

https://kuizuo.cn/blog/2024-year-end-summary/

**Comment by chaseFunny** [comment]
写的不错 👍

**Comment by huanfe1** [comment]
2025 继续加油👍

**Comment by xbhk** [comment]
牛的

**Comment by ch1lam** [comment]
新的一年继续加油👏🏻

**Comment by wangwanwang6** [comment]
X 上招聘 是哪里啊  怎么才能找到远程工作 需要哪些技术栈啊

**Comment by cmj799** [comment]
加油，人生总会有自己的出路，我现在也是很迷茫，所以干脆先等等，与其忙忙碌碌地焦虑，不如坦坦荡荡地摆烂。

**Comment by Dominic-KK** [comment]
国内的求职市场真的一言难尽了，在职牛马的生活也不好过，又穷又累
